tests/integration/gcp.py

```diff
@@ -401,36 +386,47 @@ def _upload_image(self, image_name, image):
         else:
             self.logger.info(f"Image file {image_blob.name} already exists in bucket {self._bucket}.")
 
-        images = self._compute.images()
+        images = self._compute_images
 
         blob_url = image_blob.public_url
         self.logger.info(f'Importing {blob_url} as {image_name=} into project {self.image_project}')
-        insertion_rq = images.insert(
-            project=self.image_project,
-            body={
+        config={
                 'description': 'gardenlinux',
                 'name': image_name,
-                'rawDisk': {
+                'raw_disk': {
                     'source': blob_url,
                 },
                 'labels': self._tags,
-            },
-        )
+            }
+
+        if self.config['uefi'] or self.config['secureboot']:
+            guest_os_features = {'guest_os_features': [{'type_': "UEFI_COMPATIBLE"}]}
+            config.update(guest_os_features)
+
+        if self.config['secureboot']:
+            cert_file_type = self.config['cert_file_type']
+            shielded_instance_initial_state = {
+                'shielded_instance_initial_state': {
+                    'dbs': [self._get_file_content_buffer(self.config['db_path'], cert_file_type)],
+                    'keks': [self._get_file_content_buffer(self.config['kek_path'], cert_file_type)],
+                    'pk': self._get_file_content_buffer(self.config['pk_path'], cert_file_type),
+                }
+            }
+            config.update(shielded_instance_initial_state)
```

Comment:
We should directly write `shielded_instance_initial_state` to `config` instead of using a temporary variable and the `update()` method